### ISSUE TITLE
Dialog: Accessibility - Hidden element has focusable content

### ISSUE THREAD

**Issue by DanRoddis** [primefaces/primevue#6251]
### Describe the bug

A page element with the attribute aria-hidden="true" also has a tabindex for the keyboard trap in the dialog, this appears to be to support focus when using non-assistive tech but it doesn't follow accessibility best practices and will flag as an error in any automated accessibility checkers at WCAG 2.2 A level. 

### Reproducer

https://stackblitz.com/edit/fiuumy?file=src%2FApp.vue

### PrimeVue version

3.53.0

### Vue version

4.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

**Comment by tugcekucukoglu** [comment]
Sorry, cannot catch the problem.

**Comment by kvacke** [comment]
@tugcekucukoglu This issue is visible in the Popover in the PrimeVue docs:
1. Go to https://primevue.org/popover/#selectdata
2. Open the Popover
3. Tab through the items and track the currently focused element.
4. You'll focus the element 

`<span class="p-hidden-accessible p-hidden-focusable" tabindex="0" role="presentation" aria-hidden="true" data-p-hidden-accessible="true" data-p-hidden-focusable="true" data-pc-section="lastfocusableelement"></span>`

Which should not be focusable. Should I create a separate ticket for the Popover component?
